examples/prompt_simple.cpp

```diff
@@ -0,0 +1,30 @@
+#include <cpp-terminal/version.h>
+#include <cpp-terminal/prompt.hpp>
+#include <iostream>
+
+int main() {
+    std::cout << "Running cpp-terminal version: "
+              << CPP_TERMINAL_VERSION_COMPLETE << std::endl;
+    try {
+        std::cout << "CPP-Terminal basic prompt example: \n\n";
+        switch (Term::prompt_simple("Proceed?")) {
+            case Term::Result_simple::YES:
```

Comment:
I have made with the same subset at first, but I thought that this way makes the return more easily to understand and implement. Especially because the simple version is just wrapping the blocking one with default values and already handled default options.

I can change that though, if you prefer it to be one. In the end it's just a matter of preference after all i guess.